Makefile

```diff
@@ -36,6 +36,9 @@ build:
 	CGO_ENABLED=0 go install -a -ldflags '-extldflags "-static"' sigs.k8s.io/cluster-api-provider-aws/cmd/cluster-controller
 	CGO_ENABLED=0 go install -a -ldflags '-extldflags "-static"' sigs.k8s.io/cluster-api-provider-aws/cmd/machine-controller
 
+build-test:
```

Comment:
Why not `aws-actuator`? Would naturally yield `make aws-actuator`.

Reply:
Fair point. Updated.